internal/tiltfile/tiltfile_state.go

```diff
@@ -678,11 +678,12 @@ func match(manifests []model.Manifest, matching map[string]bool) ([]model.Manife
 		var missing []string
 		for k := range matching {
 			if !matched[k] {
-				missing = append(missing, k)
+				missing = append(missing, fmt.Sprintf("%q", k))
```

Comment:
maybe `%q` `unmatchedNames` a few lines up as well, for consistency?

Reply:
i like that, broke it out into a helper